virttest/qemu_vm.py

```diff
@@ -4199,7 +4199,18 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
                     # For short period of time the status can be "inmigrate"
                     # for example when using external program
                     # (qemu commit fe823b6f87b2ebedd692ca480ceb9693439d816e)
-                    self.resume(60)
+                    # Another potential issue is when postcopy migration
+                    # fails and machine ends-up in postcopy-paused state
+                    # that is evaulated as "paused" but it's not possible
+                    # to send "cont" in such case. Let's simply log the
+                    # exception and proceed in such case, that should
+                    # raise the original exception (or VM will fail to
+                    # login if "cont" failed for different reason)
+                    try:
+                        self.resume(60)
```

Comment:
I have two concerns:

1. whether `vm.is_paused` should return `True` upon `postcopy-paused` state.

The current approach said yes but I'm not sure if it would be conflict with the definition of the method.

2. whether `vm.resume` should be performed at that time.

According to your comment, it's not possible to send `cont` in such case, therefore, I'd suggest skipping the resume at here and making a detection for this kind of failure before entering the `finally` clause.